src/main/java/racing/RacingGame.java

```diff
@@ -0,0 +1,36 @@
+package racing;
+
+import racing.view.InputView;
+import racing.view.ResultView;
+
+import java.util.ArrayList;
+
+public class RacingGame {
+    private int time;
+    private int car;
+    private ArrayList<Car> carPositions = new ArrayList<Car>();
```

Comment:
```suggestion
    private List<Car> carPositions = new ArrayList<>();
```
Java는 컴파일 시점에 타입 추론이 가능합니다. 구체적인 타입 보단 다이아몬드 연산자를 사용해주세요 😄 
https://durtchrt.github.io/blog/java/generics/9/
ArrayList를 사용할땐 다형성을 위하여 상위 타입인 List로 선언해주세요. 
https://stackoverflow.com/questions/2279030/type-list-vs-type-arraylist-in-java